src/modules/handlers/hello.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-console */
+async function hello(ctx) {
```

Comment:
What is ctx type?

Reply:
In this case, this is the type os ctx: `ctx: { body: { status: string; message: string; };`. But "Koajs Context encapsulates node's request and response objects into a single object which provides many helpful methods for writing web applications and APIs". 
To more info you can see in: https://github.com/koajs/koa/blob/master/docs/api/context.md